pkg/apis/sessions/session_state.go

```diff
@@ -64,6 +64,32 @@ func (s *SessionState) String() string {
 	return o + "}"
 }
 
+func (s *SessionState) GetClaim(claim string) string {
+	if s == nil {
+		return ""
+	}
+	switch claim {
```

Comment:
Dropping a reminder here -- Pending CHANGELOG entry, I think this one is ready to merge: https://github.com/oauth2-proxy/oauth2-proxy/pull/616

So assuming we merge that first, this will need to support the groups claim

Reply:
#616 merged! Woot! Gonna flip my review requesting that change to handle groups here